lib/controls/checkbox/index.tsx

```diff
@@ -7,18 +7,43 @@ type OwnProps = React.HTMLProps<HTMLInputElement> & {
   isStandard?: boolean;
 };
 
-function CheckboxControl({ className, isStandard, ...props }: OwnProps) {
+function CheckboxControl({
+  className,
+  isStandard,
+  checked,
+  ...props
+}: OwnProps) {
   return (
     <span
       className={classNames('checkbox-control', [
         className,
         { 'checkbox-standard': isStandard },
       ])}
     >
-      <input type="checkbox" {...props} />
-      <span className="checkbox-control-base">
-        <span className="checkbox-control-checked" />
-      </span>
+      <input type="checkbox" {...props} checked={checked} />
+      {!isStandard ? (
```

Comment:
What is `isStandard`? What cases does it appear this way?

Reply:
The checkbox component is used in two different forms right now. The default so with `isStandard` set to false, or not supplied as a prop, would be the toggles you see in places like Settings such as Share Analytics toggle.
<img width="434" alt="Screen Shot 2021-09-13 at 9 36 42 AM" src="https://user-images.githubusercontent.com/1326294/133084661-c4a98c38-af6c-4b73-a7d3-3b610de88858.png">

The `isStandard` has it show up instead as a more standard-looking checkbox such as in the Note Action panel like the screenshots included in the main description, and in the Revision Slider.